frontend/components/YotpoBottomLineHtmlReferenceContainer/index.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import YotpoBottomLineWidget from './components/YotpoBottomLineWidget';
+
+/**
+ * Creates Reference to acces DOM nodes
+ */
+class HtmlReferenceContainer extends Component {
+  static propTypes = {
+    config: PropTypes.shape().isRequired,
+    productId: PropTypes.string.isRequired,
+    product: PropTypes.shape(),
+  };
+
+  static defaultProps = {
+    product: {},
+  }
+
+  /**
+   * @param {Object} props The component props.
+   */
+  constructor(props) {
+    super(props);
+
+    this.htmlContainer = React.createRef();
+  }
+
+  /**
+   * Registers the event handler for when the user taps inside the html content.
+   */
+  componentDidMount() {
+    if (!this.props.config.yotpoAppKey) {
+      return;
+    }
+    this.htmlContainer.current.addEventListener('touchstart', this.handleTap, true);
+    this.htmlContainer.current.addEventListener('click', this.handleTap, true);
+  }
+
+  /**
+   * Removes the event handler.
+   */
+  componentWillUnmount() {
+    if (!this.props.config.yotpoAppKey) {
+      return;
+    }
+    this.htmlContainer.current.removeEventListener('touchstart', this.handleTap, true);
+    this.htmlContainer.current.removeEventListener('click', this.handleTap, true);
+  }
+
+  /**
+   * If the user tapped a link element, prevent the default behaviour.
+   * @param {Object} event The touchstart event.
+   */
+  handleTap = (event) => {
+    event.stopPropagation();
+    const aTag = event.target.closest('a') || null;
```

Comment:
aTag, divTag, and spanTag are only used in the if statement at the end. If it is decided that you don't really need that, you will not need these constants.